bool A_robot1
bool A_robot2
bool A_robot3
bool B_robot1
bool B_robot2
bool B_robot3